fvm/transactionVerifier.go

```diff
@@ -178,9 +178,9 @@ func (v *TransactionSignatureVerifier) verifyAccountSignature(
 		return nil, errors.NewInvalidPayloadSignatureError(txSig.Address, txSig.KeyIndex, err)
 	}
 
-	valid, err := v.SignatureVerifier.Verify(
+	validWOTag, err := v.SignatureVerifier.Verify(
```

Comment:
```suggestion
	validWithoutTag, err := v.SignatureVerifier.Verify(
```

I'm partial to more verbose and explicit variable names -- it took me a second to realize what this meant

Reply:
addressed in 59568b1d4f5089f0ad46d9a97716c7e2d3879de9